boms/cloud-oss-bom/RELEASING.md

```diff
@@ -71,13 +88,15 @@ run `git config credential.helper store`.
 
 ### Build the release binary with Rapid (Legacy web UI)
```

Comment:
How about making it clear that this section is not part of the current release process?

```suggestion
### (Depreacated) Build the release binary with Rapid (Legacy web UI)
```

Reply:
Or maybe we should just remove this section. There do not exists instructions on the internal team page for BOM releasing. I remember the internal page used to have some relevant instructions, but they are now out of sync too much.

Reply:
That's nice idea. Removed.